lwgenerate/general/service_account.go

```diff
@@ -0,0 +1,115 @@
+package general
```

Comment:
Package name needs to reflect what the package does. But more importantly, why creating a new package? Are we using this code in multiple places? Why not put it in lwgenerate package?

Reply:
It's now in the lwgenerate package.